app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -1,12 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import cityImage from '@images/buildings.png';
+import pvdImage from '@images/buildings.png';
+import bldImage from '@images/boulder.svg';
 import { yValue, xValue } from '@lib/circle-math';
 import { timeDiffInMinutes, timeAndDateForTimezone } from '@lib/datetime';
 import { colors } from '@lib/theme';
 import styled, { keyframes } from 'styled-components';
 import MoonPhase from '@weather/moon-phase';
 
+const getCityImage = () => {
+  if (window.location.pathname.includes('Boulder')) return bldImage;
```

Comment:
You'll want to avoid reaching into `window.location` and use props instead. This:
* minimize using browse specific calls so we can test components. You would need to mock `window.location.pathname` to run this component in a test
* allows Server Side Rendering where `window` is not available. Not necessary here but good to be aware of.

Reply:
This should be fixed now!